containers/images/core/container-assets/build/01_yum

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -e
+
+# Setting up repositories
+echo "tsflags=nodocs" >> /etc/yum.conf
+
+# enabling SCL and EPEL
+yum -y install centos-release-scl epel-release
+
+# Installing software collections and required packages
+yum -y install rh-ruby24 rh-nodejs8-nodejs \
+    rh-ruby24-ruby{,gems} rh-ruby24-rubygem-{bundler,rdoc} \
+    redhat-rpm-config hostname git postgresql qpid-tools
+
+# Installing packages that are only required during container build time
+yum -y install ${TEMP_PACKAGES} 
```

Comment:
Given these are only referenced here, I'd find it cleaner to just embed them here rather than having to look in multiple places.

Reply:
the idea was that you can override them, plus they get reverted during cleanups

Reply:
Is there a workflow to override them? You could still achieve this by putting the variable closer to the script. I get that the Dockerfile tries to serve as configuration just trying to not over complicate things.

Reply:
I could see plugins requiring additional development packages.